mmv1/third_party/terraform/resources/resource_sql_database_instance.go.erb

```diff
@@ -338,20 +353,8 @@ is set to true. Defaults to ZONAL.`,
 						"database_flags": {
 							Type:     schema.TypeList,
```

Comment:
```suggestion
							Type:     schema.TypeSet,
```

Reply:
D'oh, thanks. Can you tell I developed this locally on the generated module and then copied the changes back over? 😊